pkg/manager/member/startscript/v1/template.go

```diff
@@ -208,6 +208,59 @@ echo "/pd-server ${ARGS}"
 exec /pd-server ${ARGS}
 `
 
+// msStartScriptTplText is the pd microservice start script.
+var msStartScriptTplText = `#!/bin/sh
+
+# This script is used to start ms containers in kubernetes cluster
+
+# Use DownwardAPIVolumeFiles to store informations of the cluster:
+# https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api
+#
+#   runmode="normal/debug"
+#
+
+set -uo pipefail
+
+ANNOTATIONS="/etc/podinfo/annotations"
+
+if [[ ! -f "${ANNOTATIONS}" ]]
+then
+    echo "${ANNOTATIONS} does't exist, exiting."
+    exit 1
+fi
+source ${ANNOTATIONS} 2>/dev/null
+
+runmode=${runmode:-normal}
+if [[ X${runmode} == Xdebug ]]
+then
+    echo "entering debug mode."
+    tail -f /dev/null
+fi
+
+# Use HOSTNAME if POD_NAME is unset for backward compatibility.
+POD_NAME=${POD_NAME:-$HOSTNAME}
+pd_url="{{ .PDAddress }}"
+encoded_domain_url=$(echo $pd_url | base64 | tr "\n" " " | sed "s/ //g")
+discovery_url="${CLUSTER_NAME}-discovery.${NAMESPACE}:10261"
+until result=$(wget -qO- -T 3 http://${discovery_url}/verify/${encoded_domain_url} 2>/dev/null); do
+echo "waiting for discovery service to check PD start ..."
+sleep $((RANDOM % 5))
+done
+
+domain="${POD_NAME}.${HEADLESS_SERVICE_NAME}.${NAMESPACE}.svc{{ .FormatClusterDomain }}"
+ARGS="` + pdEnableMicroService + `--listen-addr={{ .Scheme }}://0.0.0.0:2379 \
+--advertise-listen-addr={{ .Scheme }}://${domain}:2379 \
+--backend-endpoints=${result} \
+--config=/etc/pd/pd.toml \
+"
+
+echo "starting pdms-server ..."
+sleep $((RANDOM % 10))
+echo "/pdms-server ${ARGS}"
+exec /pd-server ${ARGS}
```

Comment:
do we need to update L259 to match `/pd-server`?

Reply:
want to indicate this server is different with api server... Do you think it's unnecessary?

Reply:
as we include `${ARGS}`, I think we can use the real `/pd-server` here.